cypress/support/commands.js

```diff
@@ -1,12 +1,13 @@
-// ***********************************************
```

Comment:
i think this might be giving us a merge conflict ?

Reply:
Should I get rid of everything related to GraphQL in this branch?

Reply:
oh nvm I thought it was saying it cant merge because of conflicts but its because of the cypress tests, maybe comment them out?